src/services/ckan/portalStatistics.ts

```diff
@@ -0,0 +1,26 @@
+// SPDX-FileCopyrightText: 2024 PNED G.I.E.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+import axios from 'axios';
+import { constructCkanActionUrl } from './utils';
+
+export const makePortalStatistics = (DMS: string) => {
+  return async (): Promise<{ [key: string]: number }> => {
+    const props = ['dataset', 'catalogue', 'keyword'];
+    const counts = await Promise.all(
+      props.map(async (prop) => {
+        const url = constructCkanActionUrl(DMS, `${prop}_list`);
+        const response = await axios.get(url);
+        return response.data.result.count;
```

Comment:
Can you also retrieve the values please? We will need them for the filters

Reply:
Oh, okay